pyproject.toml

```diff
@@ -35,6 +35,7 @@ dependencies = [
     "importlib-metadata",
     "lxml>=4.9.1",
     "anytree",
+    "cffconvert",
```

Comment:
That should be a dev-dependency, I suppose?

Reply:
You are right. Moved!